### ISSUE TITLE
Release management

### ISSUE THREAD

**Issue by vincentbernat** [iovisor/bpftrace#313]
Hey!

I intend to package bpftrace for Debian. It would be nice if it was possible to get version numbers instead of just using git HEAD. This would ensure we would package a known working version and not something in the middle of a transition or with a known important bug.

**Comment by danobi** [comment]
+1, was just about to file the same issue.

**Comment by brendangregg** [comment]
I think I have access to click the release button and create one. I'm hoping to hear from @ajor about his preferences here, but if he's out of touch for a while I don't want to block Debian packaging (it's really important) and I think we can figure something out.

The biggest thing to figure out is the version number. :-) I don't think we are at 1.0, because the dust hasn't settled on the language yet. In particular is #84 and #192, which may break scripts as they are fixed, plus there are a number of known bugs we haven't fixed yet. It doesn't feel like a 1.0 release at the moment.

So what is it? All the major pieces are done for a tracer that meets people's expectations (due to what eBPF can do, we will want to add more major pieces later for a 2.0 release). I think that maybe we want to call it 0.8 now, bump it to 0.9 later when #84 and #192 are fixed, and 1.0 when we have zero or near-zero known bugs. How does that sound? (paging @ajor!).

**Comment by danobi** [comment]
I agree that it should be < 1.0 Perhaps we should start one of those fancy github kanban project things to track what issues need to be resolved for a 1.0 release?

There's a number of things I think should be fixed, the build system downloading bcc for one.

**Comment by danobi** [comment]
Whichever version number we choose, I think it'd be nice if we could run `bpftrace --version` and get a number spit out.

**Comment by vincentbernat** [comment]
I can complain here once the only missing thing is a version number. Currently, this is not the case until late next week.

**Comment by brendangregg** [comment]
@vincentbernat first release https://github.com/iovisor/bpftrace/releases/tag/v0.8
